File: backend-nest-backup/src/locations/locations.controller.ts
import { Controller, Get } from '@nestjs/common';

import { DEFAULT_CITY, DEFAULT_REGION, REGIONS } from '../common/locations';

@Controller('locations')
export class LocationsController {
  @Get()
  list() {
    return {
      defaultCity: DEFAULT_CITY,
      defaultRegion: DEFAULT_REGION,
      regions: REGIONS,
    };
  }
}

File: backend/src/main/java/com/unbora/api/domain/location/LocationsController.java
package com.unbora.api.domain.location;

import com.unbora.api.common.LocationsConstants;
import io.swagger.v3.oas.annotations.Operation;
import io.swagger.v3.oas.annotations.tags.Tag;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

@RestController
@RequestMapping("/locations")
@Tag(name = "Locations", description = "Regiões e cidades atendidas")
public class LocationsController {

    @GetMapping
    @Operation(summary = "Listar regiões e cidades padrão de Fortaleza e Ceará")
    public Map<String, Object> list() {
        return Map.of(
                "defaultCity", LocationsConstants.DEFAULT_CITY,
                "defaultRegion", LocationsConstants.DEFAULT_REGION,
                "regions", LocationsConstants.REGIONS
        );
    }
}
